mentat/commands.py

```diff
@@ -390,3 +391,53 @@ def argument_names(cls) -> list[str]:
     @classmethod
     def help_message(cls) -> str:
         return "Set a configuration option or omit value to see current value."
+
+
+class ScreenshotCommand(Command, command_name="screenshot"):
+    async def apply(self, *args: str) -> None:
+        session_context = SESSION_CONTEXT.get()
+        vision_manager = session_context.vision_manager
+        stream = session_context.stream
+        config = session_context.config
+        conversation = session_context.conversation
+        model = config.model
+
+        if "gpt" in model:
+            if "vision" not in model:
+                stream.send(
+                    "Using a version of gpt-4 that doesn't support images. Changing to"
```

Comment:
Maybe just say version of gpt? Could be using gpt-3.5

Reply:
Running the screenshot command doesn't call the model it just adds a screenshot to the conversation. The model will need to be `gpt-4-vision-preview` when the user next enters a prompt (and on all future prompts unless the run `/clear`). This is part of the reason I think this change of the model is safe: If the user doesn't want to use `gpt-4-vision-preview` they'll see the model is changed and they'll have the opportunity to change it back or quit before it is used.